bootcamp_project/Module 8/More_OOP_Concepts/VideoGame_Store_OOP_EMira1.ts

```diff
@@ -0,0 +1,78 @@
+class VideoGame {
+
+    // Static attributes
+    static availableGames: VideoGame[] = [];
+    static soldOutGames: VideoGame[] = [];
+    static totalSold: number;
+
+
+    // Instance attributes
+    title: string;
+    developer: string;
+    availableCopies: number;
+    price: number;
+    copiesSold: number;
+
+    constructor(title: string, developer: string, availableCopies: number, price: number, copiesSold: number, availableGames: string[]) {
```

Comment:
Static attributes shouldn't be present in the signature of the constructor since these are owned by the class itself and not the instance being constructed.
```suggestion
    constructor(title: string, developer: string, availableCopies: number, price: number, copiesSold: number) {
```

Reply:
Modified.